crates/task-impls/src/quorum_proposal_recv.rs

```diff
@@ -120,7 +120,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> QuorumProposalRecvTaskState<
         if let HotShotEvent::QuorumProposalRecv(proposal, sender) = event.as_ref() {
             match handle_quorum_proposal_recv(proposal, sender, event_stream.clone(), self).await {
                 Ok(Some(current_proposal)) => {
-                    self.cancel_tasks(proposal.data.get_view_number() + 1).await;
+                    tracing::error!("Cancelling tasks and voting");
```

Comment:
I think we can still call `cancel_tasks` but for the current view, not `+ 1`.

Reply:
This was just me tinkering around